### PR TITLE
update helm &containerd support

### PR DESCRIPTION
**Pull request by wenchajun**
Install the version without the plug-in

```
helm install fluentbit-operator  --create-namespace -n kubesphere-logging-system chart/fluentbit-operator/charts/k8s/
```

Install the version with the plug-in

```
helm install fluentbit-operator  --create-namespace -n kubesphere-logging-system chart/fluentbit-operator/
```
You should change the `Parser` on the input from  `docker` to `cri`  If you're running on Kubernetes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,15 +115,39 @@ kubectl apply -f manifests/logging-stack
 # kubectl kustomize manifests/logging-stack/ | kubectl apply -f -
 ```
 
+> You also can deploy Fluent Bit Operator with Helm chart.
+> Note: For the Helm-based installation you need Helm v3.2.1 or later.
+
+Install the version without the plug-in
```
Comment (wanjunlei) [inline README.md:121]: Why use `plug-in`?

File: README.md [hunk 2]
```diff
@@ -116,15 +115,39 @@ kubectl apply -f manifests/logging-stack
...
 
 ```shell
 $ curl localhost:9200/_cat/indices
 green open ks-logstash-log-2020.04.26 uwQuoO90TwyigqYRW7MDYQ 1 1  99937 0  31.2mb  31.2mb
-``` 
+```
 
 Success!
 
+### Collect containerd logs
```
Comment (wanjunlei) [inline README.md:141]: It needs to support the installation of Fluent Bit Operator on `containerd` with helm.

**Comment by FeynmanZhou** [comment]
> Install the version without the plug-in
> 
> ```
> helm install fluentbit-operator  --create-namespace -n kubesphere-logging-system chart/fluentbit-operator/charts/k8s/
> ```
> 
> Install the version with the plug-in
> 
> ```
> helm install fluentbit-operator  --create-namespace -n kubesphere-logging-system chart/fluentbit-operator/
> ```
> 
> You should change the `Parser` on the input from `docker` to `cri` If you're running on Kubernetes

@wenchajun We need to add the helm chart of Fluentbit operator to [this repo.](https://github.com/kubesphere/helm-charts/tree/master/src/main).

**Comment by FeynmanZhou** [comment]
It would be better to split this PR into two since it has two features been added.

**Pull request closed without merging**